src/ResourceManagement/RecoveryServices/RecoveryServices.Tests/packages.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
```

Comment:
The packages.config in the RecoveryServicesManagement.csproj contains:
  <package id="Hyak.Common" version="1.0.2" targetFramework="portable-net403+win+wpa81" />

So why should we take a different dependency in the test csproj?